data/Abdulwahid7027.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Shaik Abdul Wahid",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Student • Frontend developer • Open Source Enthusiast • Python • Java",
+  "avatar": "https://avatars.githubusercontent.com/u/110891341?s=96&v=4",
+    "tags": [
+    "Web Development",
+    "Html",
+    "Css",
+    "Open Source",
+    "Java",
+    "Python"
+  ],
+  "socials": [
+    { "icon": "FaGithub", "url": "https://github.com/Abdulwahid7027" },
+    { "icon": "FaTwitter", "url": "https://twitter.com/Abdulwahid7027" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/abdul-wahid-shaik-611893259/" }
+  ],
+  "links": [
+    {
+      "name": "Checkout my code contributions on GitHub",
+      "url": "https://github.com/Abdulwahid702",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Abdulwahid702",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "www.linkedin.com/in/abdul-wahid-shaik-611893259/",
```

Comment:
fix your LinkedIn URL, it should be `https://www.linkedin.com/in/abdul-wahid-shaik-611893259/`

Reply:
## Ok